config/config.yaml

```diff
@@ -103,6 +103,12 @@ branch-protection:
             contexts:
             - pull-trust-manager-verify
             - pull-trust-manager-smoke
+        issuer-lib:
+          required_status_checks:
+            contexts:
+            - lint
+            - test-e2e
+            - test-unit
```

Comment:
where is it going to find those tests?

Reply:
those are GH action tests.

Reply:
eg. https://github.com/cert-manager/issuer-lib/pull/7/files#diff-da26bf1b5f45a27fe78427b79c8711292263ae8518581763715d09723126dfabR5